bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1470,7 +1470,7 @@ private void checkStatus() {
             var commitMessage = String.join("\n", commit.message());
 
             var readyToPostApprovalNeededComment = readyForReview &&
-                    (!reviewNeeded || visitor.errorFailedChecksMessages().isEmpty()) &&
+                    visitor.hasErrors(reviewNeeded) &&
```

Comment:
Shouldn't this be negated?

Reply:
Thanks for catching it. You are right. I remember I added "!" but Somehow it disappeared...